### DISCUSSION TITLE
strand-specific RNA-Seq alignments

### DISCUSSION THREAD

**Discussion by YoshiKawahara** [GMOD/jbrowse-components#1403] [General]
Hi, I have BAM files (output of HISAT2) of strand-specific RNA-Seq alignments with XS:A:+/- tags to represent the strandedness. Are there any options to show the read alignments with different colors based on the XS tag information like the "useXS" option in JBrowse1?
Any help will be appreciated.

**Comment by cmdcolin** [comment]
Hi there,
Great to hear your interest. I have been working on customized coloring options on this branch https://github.com/GMOD/jbrowse-components/pull/1231

It now includes color by XS tag

If you have interest you can test this branch directly, it contains some experimental stuff still so it might be a little bit till it hits master branch

Here are some screenshots showing paired-end RNA-seq and XS tag coloring (the useReverseTemplate and useXS tags repectively)


![localhost_3000__config=test_data%2Fvolvox%2Fconfig json session=local-yynnfB5FF](https://user-images.githubusercontent.com/6511937/98252430-ee889680-1f47-11eb-9b84-f39448491835.png)

![localhost_3000__config=test_data%2Fvolvox%2Fconfig json session=local-yynnfB5FF (2)](https://user-images.githubusercontent.com/6511937/98252424-ecbed300-1f47-11eb-88b3-03b26449a04b.png)

**Comment by cmdcolin** [comment]
can download http://s3.amazonaws.com/jbrowse.org/code/jb2/paired_rendering/jbrowse-web-paired_rendering.zip for built version of that branch or use jbrowse cli to download it `jbrowse create /var/www/html/jb2 --url http://s3.amazonaws.com/jbrowse.org/code/jb2/paired_rendering/jbrowse-web-paired_rendering.zip`

**Comment by YoshiKawahara** [comment]
Thanks Colin! That's just what I wanted. I'll test the branch :-)
